ansible/roles/test/files/ptftests/mtu_test.py

```diff
@@ -143,11 +192,55 @@ def check_ip_mtu(self):
 
     #---------------------------------------------------------------------
 
+    def check_ipv6_mtu(self):
+        '''
+        @summary: Check unicast IPv6 forwarding in DUT works for MAX MTU.
+        '''
+        ipv6_src = self.src_host_ipv6
+        ipv6_dst = self.dst_host_ipv6
+        src_mac = self.dataplane.get_mac(0, self.src_ptf_port_list[0])
+
+        pkt = simple_ipv6ip_packet(pktlen=self.pktlen,
```

Comment:
I think this is to generate a IPv6 in IPv6 packet. It would be better to generate a pure IPv6 packet using function like `simple_udpv6_packet` in the ptf.testutils module.
https://github.com/p4lang/ptf/blob/master/src/ptf/testutils.py#L1676

Reply:
Well, I use `simple_udpv6_packet` instead of `simple_ipv6ip_packet`, Thanks!